spec/content.spec.js

```diff
@@ -649,6 +649,7 @@ describe('Content TestCase', function () {
                 fireEvent(target, 'keydown', {
                     keyCode: MediumEditor.util.keyCode.BACKSPACE
                 });
+                this.el.innerHTML = stripLinebreak(this.el);
                 expect(this.el.innerHTML).toBe('<p>lorem ipsum</p><ul><li></li><li>lorem ipsum</li></ul>');
```

Comment:
could you use a `matches` test here instead of `toBe`?  that will let you use a regex and not have to modify the actual element for the comparison.  I've use the regex thing in a bunch of tests in the repo.

Reply:
Sorry, i meant `toMatch`, put too much trust in my memory.  Here's an [example](https://github.com/yabwe/medium-editor/blob/5.23.2/spec/core-api.spec.js#L158-L159)